modules/serverless-kn-cli-plugins.adoc

```diff
@@ -5,6 +5,6 @@
 [id="serverless-kn-plugins_{context}"]
 = kn plug-ins
 
-The Red Hat distribution of the CLI command `kn` contains extensions of `kn` known as plug-ins. The main use of plug-ins is to extend `kn` by providing additional commands on {ServerlessProductName}, such as for the Kafka stack API. The `kn` plug-ins are used in the same way as the main `kn` functionality.
+The `kn` CLI supports the use of plug-ins. Plug-ins enable you to extend the functionality of your `kn` installation by adding custom commands and other shared commands that are not part of the core distribution of `kn`. `kn` CLI plug-ins are used in the same way as the main `kn` functionality.
```

Comment:
Do you mean not using the term "plug-ins"? I don't think so since the thing we're talking about is literally called a plugin, e.g. https://github.com/knative-sandbox/kn-plugin-source-kafka

Reply:
Think I just got what you meant. Maybe:
"The `kn` CLI supports the use of plug-ins, which enable you to extend the functionality of your `kn` installation by adding custom commands and other shared commands that are not part of the core distribution."
?

Reply:
@abrennan89, oh, I just though that ending a sentence with _plug-in_ and starting the next with _Plug-in_ is a bit jarring to read.

Reply:
@abrennan89 but I can also just merge this, it isn't a blocker at all.